index.html

```diff
@@ -400,29 +400,9 @@
             <section>
                 <h2>Firing events using the <code>PointerEvent</code> interface</h2>
                 <p>To <dfn>fire a pointer event name e</dfn> means to <dfn>fire an event named e</dfn> as defined in [[!DOM4]] with an event using the <a>PointerEvent</a> interface whose attributes are set as defined in <a href="#pointerevent-interface"><code>PointerEvent</code> Interface</a>.</p>
+                <p>For any pointer event initialize the attributes of the event according to the <a href="#h-pointer-event-types">Pointer Event types</a> section.</p>
```

Comment:
I think would be clearer if merged the previous paragraph since it's really a continuation of the theme of initializing the attributes.  Maybe: "... whose attributes are set as defined in PointerEvent interface, and in Pointer Event types for the type e.".

Also while you're editing that line, can you fix the typo: "fire a pointer event name e" -> "fire a pointer event named e"?